### PR TITLE
Milestone 06 : Views

### PR DESCRIPTION
**Pull request by FaranoshAmini**
## In this pull request, I have implemented the following changes:

- [x]  Add the Views associated with the Models and Controllers already created for the rails-blog.
- [x]  Implement the designs from the sneak peek wireframes.
- [x]  Use methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,21 @@
...
+Likes: <%=post.likes_counter %></p>
+</div>
+<div>
+<% post.recent_comments.each do |comment| %>
+ <div class="bio"><%= comment.author.name %>:  <%= comment.text %></div>
+<% end %>
+</div>
+<% end %>
+<button class="btn"><a href="">Pagination</a></button>
+<div>
```
Comment (zieeco) [inline app/views/posts/index.html.erb:21]: ## POST.RB FILE


- Hey, your app throws an error when trying to create a post for a user this is happening as you're calling this [method](https://github.com/FaranoshAmini/rails-blog/blob/6ebf6c188e4ca1f5f8043fc41bed2b460729c895/app/models/post.rb#L7) that does not exist in your `post.rb` file.


- To fix this, kindly create a private method in your `post.rb` file model to increment the post counter

```ruby
private
  def post_counter_update
    author.increment!(:posts_counter) // You can implement this anyway you want.
  end
```
*This should fix the error*

> **[OPTIONAL]** Also, kindly add some data in your seeds file so you can just populate the database with this data using the `rails db:seed` command.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @FaranoshAmini,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice views.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @FaranoshAmini 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FaranoshAmini** [comment]
@codepantha, and @zieeco, Thanks for your kind review.

**Pull request merged**